app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/attachments_list.scala.html

```diff
@@ -0,0 +1,54 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.models.StoredAttachment
+
+@import uk.gov.hmrc.tariffclassificationfrontend.utils.Dates
+
+@(heading: String, attachments: Seq[StoredAttachment], showUploadedBy : Boolean  = false)(implicit messages: Messages)
+
+    <h3>@heading</h3>
+
+    @if(attachments.nonEmpty){
+        <table>
+            <thead>
+                <tr>
+                    <th scope="col">File name</th>
+                    @if(showUploadedBy) { <th scope="col">Uploaded By</th>}
+                    <th class="attachment-table-date" scope="col">Date</th>
+                </tr>
+            </thead>
+            <tbody>
+                    @for(file <- attachments) {
+                        <tr>
+                            <td class="w-50"><a href="@file.url" target="_blank">@file.fileName</a></td>
+                            @if(showUploadedBy) {
+                                <td>
+                                @if(file.operator.isDefined){
+                                     @file.operator.get.name
+                                }else{
+                                     Undefined
```

Comment:
I presume Undefined is from the prototype? I would say "Unknown" or "N/A" would be more user friendly

Reply:
If its from the prototype though thats fine

Reply:
It's not from the prototype, then we could change it for Unknown potentially